firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -314,7 +322,8 @@ private void persistFid(String fid) throws FirebaseInstallationsException {
   }
 
   /** Registers the created Fid with FIS servers and update the shared prefs. */
-  private Void registerAndSaveFid(PersistedInstallationEntry persistedInstallationEntry)
+  private Void registerAndSaveFid(
+      PersistedInstallationEntry persistedInstallationEntry, String iidToken)
```

Comment:
instanceIdMigrationAuth?

Please make a brief comment int he JavaDoc, that the instanceIdMigrationAuth is a wildcard-scoped ("*") IID-Token created by IID/FCM's endpoint /register3.